jib-core/src/integration-test/java/com/google/cloud/tools/jib/registry/ManifestPullerIntegrationTest.java

```diff
@@ -65,13 +79,31 @@ public void testPull_v21() throws IOException, RegistryException {
   @Test
   public void testPull_v22() throws IOException, RegistryException {
     RegistryClient registryClient =
-        RegistryClient.factory(EventHandlers.NONE, "gcr.io", "distroless/java", httpClient)
+        RegistryClient.factory(EventHandlers.NONE, "gcr.io", "distroless/base", httpClient)
             .newRegistryClient();
```

Comment:
qq you may have covered this already but why do we need to replace `distroless/java` with `distroless/base`? Is there anything blocking us from using a previous tag from [`gcr.io/distroless/java`](https://pantheon.corp.google.com/gcr/images/distroless/global/java).

Reply:
Mostly for consistency and convenience, since we're already pinning a V22 manifest version of `distroless/base` image for use by `BlobCheckerIntegrationTest` and  `BlobPullerIntegrationTest`. 

If there are concerns with switching the image here, `gcr.io/distroless/java:latest` will actually still work for this test, since it has not been updated since April 2022. So we could also leave this test unchanged, or pin it to that version by SHA. WDYT?

Reply:
oh gotcha, thank you! Making the base images consistent is a good idea.